### DISCUSSION TITLE
Html conversion malformed

### DISCUSSION THREAD

**Discussion by jeromep01** [gotenberg/gotenberg#470] [Q&A]
Hi,

i've generated a HTML file with 595px/842px dimensions. But when i print, the result does not take the whole A4 page.

What am i doing wrong ?

Can you help me ?
[_Templates.zip](https://github.com/gotenberg/gotenberg/files/9010551/_Templates.zip)

**Comment by gulien** [comment]
Hello @jeromep01,

The form fields `paperWidth` and `paperHeight ` are what you are looking for. See https://gotenberg.dev/docs/modules/chromium#routes for more details.
